engines/hadesch/module.mk

```diff
@@ -0,0 +1,53 @@
+MODULE := engines/hadesch
+
+MODULE_OBJS = \
+    detection.o \
```

Comment:
You would need to move `detection.o` to `DETECTION_OBJS`

Reply:
done